### ISSUE TITLE
ask for tensoboard results

### ISSUE THREAD

**Issue by kevin-why-2020** [FT-ZHOU-ZZZ/CMTA#5]
Please show your training tensorboard log results in BLCA dataset, I suspect the whole training process is overfit, thank you. Since i follow the all hyperparameters and the same dataset, but found train loss is from high to low but val loss is from low to high.

**Comment by Curry30Messi** [comment]
I also have the same question

**Comment by Curry30Messi** [comment]
Perhaps because there is no cross-domain testing, there is a risk of overfitting itself
